pages/common/steam.md

```diff
@@ -0,0 +1,24 @@
+# steam
+
+> Video game platform by Valve.
+> More information: <https://developer.valvesoftware.com/wiki/Command_Line_Options#Steam_.28Windows.29>.
+
+- Launch Steam and show debug messages in the terminal:
+
+`steam`
+
+- Launch Steam with a debug console tab included in-app:
```

Comment:
This is still correct, is it not?
```suggestion
- Launch Steam and enable the debug console tab:
```

Reply:
yes. however, i want to make it clear that the debug console is a tab in the steam app, to contrast it with the earlier example that gives you debug info in the bash shell you ran the command from. does something like:

`Launch Steam and enable its in-app debug console tab:`

look ok to you?

Reply:
committed this and the previous change